src/api/routes/rosetta/network.ts

```diff
@@ -95,7 +95,9 @@ export function createRosettaNetworkRouter(db: DataStore, chainId: ChainID): Rou
 
     const refUrl = ReferenceNodes[getRosettaNetworkName(chainId)];
```

Comment:
@raynaudoe  Rosetta tests were failing due to this, because in tests `refUrl` resolves to `testnet.stacks.co`.  
cc: @zone117x .